src/main/java/domain/LottoRankData.java

```diff
@@ -0,0 +1,75 @@
+package domain;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public enum LottoRankData implements CountWinnerStrategy {
+    FIRST_WINNER(2000000000, 6, () -> getCountWinner(6)),
+
+    SECOND_WINNER(1500000, 5, () -> getCountWinner(5)),
+
+    THIRD_WINNER(50000, 4, () -> getCountWinner(4)),
+
+    FORTH_WINNER(5000, 3, () -> getCountWinner(3)),
+
+    BONUS_SECOND_WINNER(30000000, 5, () -> getCountWinner(1));
+
+    private int prize, match;
+    private static int bonusBallCount;
+    static List<Integer> countsOfWinningResultConvertList;
+    private CountWinnerStrategy countWinnerStrategy;
+    private final static int BONUS_MATCH = 1;
```

Comment:
상수는 최상단에 작성해주세요.

Class (static) variables: First the public class variables, then the protected, and then the private.

Instance variables: First public, then protected, and then private.

Constructors

Methods: These methods should be grouped by functionality rather than by scope or accessibility. For example, a private class method can be in between two public instance methods. The goal is to make reading and understanding the code easier.